lua/gatekeeper/policylib.lua

```diff
@@ -318,3 +318,46 @@ function lpm_para_estimate(ipv4_file)
 
 	return num_rules, num_tbl8s
 end
+
+local function lpm6_add_tbl8s(ip6_addr, prefix_len, prefixes)
+	local prefix
+	local masked_ip
+	local depth = 24
+	local ret = 0
+
+	-- For a prefix with length longer than 24, one tbl8
+	-- is needed every 8 bits. If the prefix length is not
+	-- a multiple of 8, then prefix expansion will be performed
+	-- on that tbl8 entry. More details can be found on
+	-- the addtion description in DPDK LPM6 library:
+	-- https://doc.dpdk.org/guides/prog_guide/lpm6_lib.html#addition
+	while (depth < prefix_len)
+	do
+		masked_ip = lpmlib.ip6_mask_addr(ip6_addr, depth)
+		prefix = masked_ip .. "/" .. depth
+		if prefixes[prefix] == false then
```

Comment:
When not set, `prefixes[prefix]` is `nil`. Just use the expression `not prefixes[prefix]` instead.